packages/shared/package.json

```diff
@@ -112,6 +112,7 @@
     "lazysizes": "^5.3.2",
     "linkifyjs": "^2.1.9",
     "lodash.clonedeep": "^4.5.0",
-    "node-fetch": "^2.6.6"
+    "node-fetch": "^2.6.6",
+    "react-swipeable": "^6.2.0"
```

Comment:
What's the package size? Have you checked if there are any lighter alternatives?

Reply:
Unpacked Size
153 kB

Can have a look what else is out there :)

Reply:
Please fix the build issue so we can see how much payload it adds to our bundles

Reply:
@idoshamun Having some issues with React not liking this hook implementation.
Do you have any guidance on where to look for this issue?